### PR TITLE
slack/AddReadme: Adding a Readme file

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE PR

Jira Ticket: [MAT-3836](https://jira.cms.gov/browse/MAT-3836)
(Optional) Related Tickets:

### Summary

This ticket is just for adding a README.

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [X] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [X] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+This is a Spring Boot micro-service which is responsible for operations associated with the MADiE Measure Micro Frontend.
+
+# Running Locally
+
+The application can be run as a spring-boot application. It requires a locally running MongoDb instance.
+
+The MongoDb instance can be run with 
+
+```
+docker compose down --remove-orphans && docker volume prune && docker compose build --no-cache && docker compose up --force-recreate --build madie-mong
```
Comment (nmorasb) [inline README.md:12]: missing "o" at the end of the line for "madie-mongo" - command will likely fail without that

**Review by adongare — APPROVED** [review]
